packages/ubuntu_localizations/lib/src/l10n/ubuntu_hu.arb

```diff
@@ -1,5 +1,7 @@
 {
   "@@locale": "hu",
   "countryCode": "HU",
-  "languageName": "Magyar"
+  "@countryCode": {},
+  "languageName": "magyar",
```

Comment:
Languages in the Ubuntu Desktop Installer and Ubuntu WSL setup wizard are expected to be capitalized, and "Magyar" is capitalized in Ubiquity too. I've proposed an update in Weblate.